### PR TITLE
Play Button Design fix for issue #136

### PR DESCRIPTION
**Pull request by KrystalJean**
## What does this PR do?

1. Changed the green play, next round, and retry buttons to a pokeball button.
2. Added the new screenshots to the README.md file.

## Screenshots of changes made to the webapp.

play
next-round
retry

## Related PRs and Issues

Closes issue #136 .

### Have you updated the README.md file?

- [x] yes
- [ ] no

### Have you made any big breakthrough change?

- [x] yes
- [ ] no

### Is the code beautified?

- [x] yes
- [ ] no

### Is the code clear to understand with proper variable naming and comments?

- [x] yes
- [ ] no

### CODE REVIEW

**Comment by Aatmaj-Zephyr** [comment]
Please add the relevant screenshots to the README.md file

**Review by Aatmaj-Zephyr — APPROVED** [review]
Approved

**Comment by KrystalJean** [comment]
Hi, I just wanted say sorry that I missed your comment yesterday about adding screenshots to the README.md file, but I completed that now, and added it to the pull request.  : )

File: README.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ This is the pokemon version of the hangman game, with names of pokemon as words.
 
 Gotta catch 'em all!
 
-<img width="1503" alt="image" src="https://user-images.githubusercontent.com/83284294/194230811-a339fcf9-27af-41c2-b132-7479c18979c7.png">
-
+![play](https://user-images.githubusercontent.com/44277065/194386077-b0a5e21e-9583-4b5b-a682-47024c3c6ed7.png)
```
Comment (Aatmaj-Zephyr) [inline README.md:7]: Thanks

**Pull request merged**